scripts/core/editor3/reducers/tests/suggestion_setup.jsx

```diff
@@ -0,0 +1,141 @@
+import {EditorState, convertFromRaw} from 'draft-js';
+import reducer from '../suggestions';
+
+function prepareRawContent(rawContent) {
+    const defaultBlock = {
+        entityRanges: [],
+        inlineStyleRanges: [],
+        depth: 0,
+        type: 'unstyled',
+        text: '',
+        data: {},
+    };
+
+    let newRawContent = {
+        blocks: [],
+        entityMap: {},
+        ...rawContent,
+    };
+
+    let blocks = newRawContent.blocks.map((block) => ({
+        ...defaultBlock,
+        ...block,
+    }));
+
+    newRawContent.blocks = blocks;
+
+    return newRawContent;
+}
+
+export function getInitialEditorState(rawContent) {
+    const content = convertFromRaw(prepareRawContent(rawContent));
+
+    return EditorState.createWithContent(content);
+}
+
+export function applySelection(editorState, startBlockIndex, startOffset, endBlockIndex, endOffset) {
```

Comment:
This seems simpler to me
https://github.com/superdesk/superdesk-client-core/blob/687d4259f6bbc619d62a6961d79f744308c4cd88/scripts/core/editor3/components/tests/multipleHighlights.spec.jsx#L78

Reply:
I think the current solution is fine, it cover also the selection of all blocks.